exercises/concept/pacman-rules/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+# Instructions
+
+In this exercise, you need to translate some rules from the classic game Pac-Man into Haskell functions.
+
+You have four rules to translate, all related to the game states.
+
+> Don't worry about how the arguments are derived, just focus on combining the arguments to return the intended result.
+
+## 1. Define if Pac-Man eats a ghost
+
+Define the `eatsGhost` function that takes two arguments (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a `Boolean` value if Pac-Man is able to eat the ghost. The function should return `True` only if Pac-Man has a power pellet active and is touching a ghost.
```

Comment:
```suggestion
Define the `eatsGhost` function that takes two arguments (_whether Pac-Man has a power pellet active_ and _whether Pac-Man is touching a ghost_) and returns a `Boolean` value indicating whether Pac-Man is able to eat the ghost. The function should return `True` only when Pac-Man has a power pellet active and is touching a ghost.
```
This may be splitting hairs. Not sure whether it really is clearer to people other than me.

Reply:
If we make the change for one function it should be the same on the following three.

Reply:
Yes. I have a patch ready; if it is decided that _whether_ & _when_ are prefered I can push it.

@ErikSchierboom Do I understand correctly that you meant this suggestion?